src/sentry/web/frontend/base.py

```diff
@@ -262,7 +262,7 @@ def handle_not_2fa_compliant(self, request, *args, **kwargs):
         redirect_uri = self.get_not_2fa_compliant_url(request, *args, **kwargs)
         return self.redirect(redirect_uri)
 
-    def get_no_permission_url(request, *args, **kwargs):
+    def get_no_permission_url(self, request, *args, **kwargs):
```

Comment:
```suggestion
    def get_no_permission_url(self, *args, **kwargs):
```

🤷 we don't use request right?

Reply:
hm, let me add that back lol

Reply:
Oh wait, nah it should stay.. for whatever reason